### PR TITLE
Creating venues details 

### PR DESCRIPTION
**Pull request by spokrovskiy**
We created tests and a page for venue details

### CODE REVIEW

File: spec/features/venues/view_venue_details_spec.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+feature 'View details of the venue' do
```
Comment (jraenorris) [inline spec/features/venues/view_venue_details_spec.rb]: In terms of using terminology that's widely used, calling this a "Venue show page" would also be accurate and possibly more explicit for future devs

File: spec/features/venues/view_venue_details_spec.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  let!(:venue) { FactoryGirl.create(:venue) }
+
+  scenario 'When a user clicks on the venue name see venue details.' do
+    visit venues_path
+
+    click_link(venue.name)
+
+    expect(page).to have_content(venue.name)
```
Comment (jraenorris) [inline spec/features/venues/view_venue_details_spec.rb]: I would check for a couple details of the venue that would not also be on the venue index page. Throw in an expectation for one of the venue details besides name here

File: spec/features/venues/view_venue_details_spec.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  end
+
+  scenario 'When user vists page should see venue details' do
```
Comment (jraenorris) [inline spec/features/venues/view_venue_details_spec.rb]: Look at this sentence a little more. Could you make it more explicit? For example 'when user visits page' -- what page? The index page? The show page?

**Comment by jraenorris** [comment]
@spokrovskiy @arbertheo @nvanselow 

Looking good so far. Made a couple comments for you to address. Keep it up and ping me when you're ready for a second pass

**Comment by spokrovskiy** [comment]
We clarified the show page (instead of generic page). We added additional checks to make sure we were correctly moving to the details page from the index page.

**Pull request merged**